lib/attendees.js

```diff
@@ -0,0 +1,19 @@
+const { getCollection } = require("./database");
+
+function getAttendeeCollection() {
+  return getCollection("attendees");
+}
+
```

Comment:
You don't need `async` and `await` here if you directly return the result.
Same effect when you remove both